bin/start-systemtest

```diff
@@ -0,0 +1,206 @@
+#!/bin/bash
+
+##
+
+## checking args
+if [ $# -ne 3 ]
+then
+  echo "Usage: $0 USER_EMAIL USER_PASSWORD DISPATCHER_ADDRESS"
+  exit 1
+fi
+
+# args
+readonly USER_EMAIL=${1}
+readonly USER_PASSWORD=${2}
+readonly DISPATCHER_ADDRESS=${3}
+
+readonly SANDBOX=$(cd "$(dirname "${0}")" && pwd)
+readonly SUBMIT_TASK_SCRIPT=${SANDBOX}/submit-task
+readonly GET_TASK_BY_ID_SCRIPT=${SANDBOX}/get-task-by-id
+readonly GET_TASK_LINKS_SCRIPT=${SANDBOX}/get-task-links
+readonly SYSTEMTEST_MD5SUM_RESULTS_FILE=${SANDBOX}/../resources/systemtest-md5sum-results
+
+get_timestamp(){
+  TIMESTAMP=$(date +"%T")
+  echo "${TIMESTAMP}"
+}
+
+get_timesleep(){
+  local TEST_VERSION=${1}
+
+  if [ ${TEST_VERSION} == 'nop' ]; then
+    echo "30"
+  elif [ ${TEST_VERSION} == 'nop-fake-files' ]; then
+    echo "60"
+  elif [ ${TEST_VERSION} == 'nop-download-files' ]; then
+    echo "120"
+  else
+    echo "300"
+  fi
+}
+
+get_task_by_id(){
+  local ID=${1}
+  local TASK=$(bash ${GET_TASK_BY_ID_SCRIPT} ${USER_EMAIL} ${USER_PASSWORD} ${ID} ${DISPATCHER_ADDRESS})
+
+  echo "${TASK}"
+}
+
+get_dataset(){
+  local TASK=${1}
```

Comment:
this is complicate. please, add an internal doc explaining this

Reply:
done

Reply:
please, give an example based on a value to TASK and the generated DATASET value

Reply:
done